dev/Code/CryEngine/RenderDll/Common/RenderThread.h

```diff
@@ -234,10 +234,11 @@ struct SRenderThread
 #ifdef USE_LOCKS_FOR_FLUSH_SYNC
     int m_nFlush;
     CryMutex                            m_LockFlushNotify;
-    CryConditionVariable    m_FlushCondition;
 #if defined(USE_HANDLE_FOR_FINAL_FLUSH_SYNC)
+    HANDLE m_FlushCondition;
     HANDLE m_FlushFinishedCondition;
 #else
+    CryConditionVariable    m_FlushCondition;
```

Comment:
We were seeing cases where the render thread was not waking immediately after the main thread signalled it. Previously this was using a heavy-weight condition variable, which was overkill and causing the inefficiency. Changed it to use a lightweight Event instead.